src/main/groovy/org/mockito/release/internal/gradle/TravisPlugin.java

```diff
@@ -35,7 +37,7 @@ public void apply(Project project) {
         boolean isPullRequest = pr != null && !pr.trim().isEmpty() && !pr.equals("false");
 
         conf.getBuild().setCommitMessage(System.getenv("TRAVIS_COMMIT_MESSAGE"));
-        conf.getBuild().setBranch(System.getenv("TRAVIS_BRANCH"));
+        gitStatus.setBranchName(System.getenv("TRAVIS_BRANCH"));
```

Comment:
This is a little bit risky. If some other plugin requests the branch during Gradle's configuration time, and before travis plugin is applied then some objects might get wrong value of the branch. I suggest to apply my earlier suggestion and get rid of setBranchName method.